xdocs/usermanual/functions.xml

```diff
@@ -1765,6 +1765,8 @@ However some variables are defined internally by JMeter. These are listed below.
 <li><code>JMeterThread.last_sample_ok</code> - whether or not the last sample was OK - <code>true</code>/<code>false</code>.
 Note: this is updated after PostProcessors and Assertions have been run.
 </li>
+<li><code>__jmv_SAME_USER</code> - is the same user for each iteration ? <code>true</code>/<code>false</code>. Mainly impacts clean cookie, cache data and new connection</li>
```

Comment:
Is this to fix some older code?

Reply:
> _Is this to fix some older code?_
Yes, when i search this variable "__jmv_SAME_USER" in the documentation, i don't see it. 
So i add this line about the variable __jmv_SAME_USER.
And i add the new one variable  "__jmv_THREAD_START_TIME_ITERATION"